packages/botonic-cli/src/commands/new.ts

```diff
@@ -86,7 +87,11 @@ Creating...
         dir: userProjectName,
       })
     } catch (e) {
-      throw new Error(`Error downloading ${selectedProjectName}: ${String(e)}`)
+      const error = `Downloading Project: ${selectedProjectName}: ${String(e)}`
+      trackError('Created Botonic Bot CLI', {
```

Comment:
Created or Creating?

Reply:
Or we can just name it "Failed creation ..." or "Not created..." and drop the initial "Error:" injected in trackError